src/main/resources/org/broadinstitute/dsde/sam/liquibase/changesets/20201029_resource_type_fields.xml

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog logicalFilePath="dummy"
+                   xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+                   xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+                   xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                   xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.4.xsd">
+
+    <changeSet logicalFilePath="dummy" author="dvoet" id="resource_type_fields">
+        <addColumn tableName="sam_resource_type">
+            <column name="owner_role_name" type="VARCHAR" defaultValue="">
+                <constraints nullable="false"/>
+            </column>
+            <column name="reuse_ids" type="BOOLEAN" defaultValue="false">
+                <constraints nullable="false"/>
+            </column>
+        </addColumn>
```

Comment:
These columns are in config but not the db. Need to add them to the db so we can do proper comparisons between config and db.